guillotina_amqp/tests/test_api.py

```diff
@@ -114,3 +121,56 @@ async def test_info_task_filtered_response(
         )
         assert status == 200
         assert "job_data" not in resp
+
+
+async def join_amqp_worker(task_id=None):
```

Comment:
just minor comment, if someone is using the `join_amqp_worker` providing a `task_id` maybe better just change it for using the new `wait_for_task`, would make this function clearer IMO.